#!/usr/bin/env python

PACKAGE = "anomaly_detection"

from dynamic_reconfigure.parameter_generator_catkin import *
	
gen = ParameterGenerator()

#       Name     	   Type Level Description        	 Default Min  Max
gen.add("battery_runtime", int_t, 0, "Battery runtime in seconds", 120, 1, 7200)

gen.add("new_battery_level", int_t, 0, "New battery level", 100, 0, 100)

exit(gen.generate(PACKAGE, "battery_simulator", "BatterySimulator"))
